### PR TITLE
Minor manifest errors

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
The table in the new **Namespace values** section isn't great looking, but its the least bad way I found for to format this info and express it concisely.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a4fd45f](https://github.com/OfficeDev/office-js-docs-pr/commits/a4fd45fb3592d5e9df2760b7f1682308c27a4318)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/officeapp.md](https://github.com/OfficeDev/office-js-docs-pr/blob/minor-manifes-errors/docs/reference/manifest/officeapp.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/minor-manifes-errors/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002042310558629-1578%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=KffUg3XQuW8owQQ4Zht3ypgcZnXWDPDSXh2ofb8z1m4%3D&st=2020-02-04T23%3A07%3A06Z&se=2020-03-06T23%3A12%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/versionoverrides.md [hunk 1]
```diff
@@ -13,18 +13,29 @@ The root element that contains information for the add-in commands implemented b
 
 |  Attribute  |  Required  |  Description  |
 |:-----|:-----|:-----|
-|  **xmlns**       |  Yes  |  The schema location, which must be `http://schemas.microsoft.com/office/mailappversionoverrides` when `xsi:type` is `VersionOverridesV1_0`, and `http://schemas.microsoft.com/office/mailappversionoverrides/1.1` when `xsi:type` is `VersionOverridesV1_1`.|
+|  **xmlns**       |  Yes  |  The VersionOverrides schema namespace. The allowed values vary depending on the value of this `<VersionOverrides>` element's **xsi:type** value and the **xsi:type** value of the parent `<OfficeApp>` element. See [Namespace values](#namespace-values) below.|
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/versionoverrides.md]:
```suggestion
|  **xmlns**       |  Yes  |  The VersionOverrides schema namespace. The allowed values vary depending on this `<VersionOverrides>` element's **xsi:type** value and the **xsi:type** value of the parent `<OfficeApp>` element. See [Namespace values](#namespace-values) below.|
```

File: docs/reference/manifest/versionoverrides.md [hunk 2]
```diff
@@ -13,18 +13,29 @@ The root element that contains information for the add-in commands implemented b
...
 |  **xsi:type**  |  Yes  | The schema version. At this time, the only valid values are `VersionOverridesV1_0` and `VersionOverridesV1_1`. |
 
+### Namespace values
+
+The following table shows the required value of the **xmlns** value depending on the value of this `<VersionOverrides>` element's **xsi:type** value and the **xsi:type** value of the parent `<OfficeApp>` element.
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/versionoverrides.md]:
```suggestion
The following table shows the required value of the **xmlns** value depending on this `<VersionOverrides>` element's **xsi:type** value and the **xsi:type** value of the parent `<OfficeApp>` element.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4d5fac5](https://github.com/OfficeDev/office-js-docs-pr/commits/4d5fac581adea80eca433ca053992234dcbc05c2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/officeapp.md](https://github.com/OfficeDev/office-js-docs-pr/blob/minor-manifes-errors/docs/reference/manifest/officeapp.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/minor-manifes-errors/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002042342139966-1578%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=f8ATcf7VIV3Jx28bGvV3dC7DrTawOgTfaMxkd%2BlDxnQ%3D&st=2020-02-04T23%3A38%3A23Z&se=2020-03-06T23%3A43%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f55dd84](https://github.com/OfficeDev/office-js-docs-pr/commits/f55dd84716d20157749eee2325dd8bf2e6c5d658)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/officeapp.md](https://github.com/OfficeDev/office-js-docs-pr/blob/minor-manifes-errors/docs/reference/manifest/officeapp.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/versionoverrides.md](https://github.com/OfficeDev/office-js-docs-pr/blob/minor-manifes-errors/docs/reference/manifest/versionoverrides.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002042346538053-1578%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bhpB81gEGkLkLgMvK4JKA4Sh8QumzCdAXr1M7s9%2BrOw%3D&st=2020-02-04T23%3A43%3A01Z&se=2020-03-06T23%3A48%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**